pkgs/applications/networking/browsers/firefox-pmos-mobile/default.nix

```
@@ -0,0 +1,70 @@
+{ browser
```

```suggestion
{ firefox
```
No other browser will work.